docs/theming.md

```diff
@@ -185,9 +185,174 @@ extension MyViewController {
 ```
 <!--</div>-->
 
+## Migration guide: Themers to Theming extensions
+
+This migration guide covers the typical migration path from Themer usage to Theming extensions.
+
+Theming extensions are discussed in detail above. Themers are discussed in the [Themers](#themers)
+section. Themers will eventually be deprecated and deleted. Theming extensions are the recommended
+replacement.
+
+In general, every component's `Themer` targets will gradually be replaced by a single `Theming`
+extension target. This includes:
+
+- ColorThemer
+- FontThemer
+- ShapeThemer
+- TypographyThemer
+- Component Themers (e.g. CardThemer)
+
+Some Theming extensions are still in Beta. To add a Beta Theming extension to your project you will
+need to follow the instructions in the [Beta components](../contributing/beta_components.md)
+documentation. Non-Beta Theming extensions can be added to your project like any other target
+provided in the MDC repo.
+
+Some components do not have a Theming extension yet. We are prioritizing the remaining Theming
+extensions in https://github.com/material-components/material-components-ios/issues/7172.
+
+#### Typical migration
+
+When migrating from Themers to a Theming extension the first thing to understand is that a Theming
+extension will apply *all* of the Material Theming subsystems (Color, Typography, Shape) to a given
+component. If you were previously relying on the ability to apply only one subsystem (e.g. Color)
+to a component, please file a
+[feature request](https://github.com/material-components/material-components-ios/issues/new/choose)
+with a code snippet of your existing use case.
+
+Generally-speaking, the migration from a subsystem Themer to a Theming extension will involve the
```

Comment:
Nit: Remove this hyphen.  You are speaking generally. Even better, remove everything before the comma.